RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# Releases
+This page describes the release process and the currently planned schedule for upcoming releases as well as the respective release shepherd. Release shepherds are chosen on a voluntary basis.
+
+## Release Schedule
+
+Release cadence of first pre-releases being cut is 12 weeks.
+
+| release series | date (year-month-day) | release shepherd              |
+|----------------|-----------------------|-------------------------------|
+| v0.26          | 2023-05-31            | Josh Abreu (Github: @gotjosh) |
```

Comment:
Any news on an actual release? This date looked promising. ;-)

I'd really like to use #3226

Reply:
I'm currently waiting on #3353 as I know this will be quite a big release, I'd love for users to to try and enable that as well. We're closer now than ever ❤️

Reply:
Doesn't look like there was any progress on #3353 in the past 7 weeks. Do we really need to wait for that PR to be included in the next release?